docs/using.md

```diff
@@ -323,40 +226,69 @@ authenticated users. The following procedure describes how to mount this folder.
 The Minishift VM is exposed to the host system via a host-only IP address, that can be obtained
 with the `minishift ip` command.
 
-<a name="persistent-volumes"></a>
-## Persistent volumes
-
-Minishift supports [PersistentVolumes](https://docs.openshift.org/latest/dev_guide/persistent_volumes.html)
-of type `hostPath`. These PersistentVolumes are mapped to a directory inside the Minishift VM.
-
-The MiniShift VM boots into a tmpfs, so most directories will not be persisted across reboots (for example, when you use `minishift stop`).
-However, MiniShift is configured to persist files stored under the following host directories:
-
-* `/data`
-* `/var/lib/minishift`
-* `/var/lib/docker`
-
-Here is an example PersistentVolume config to persist data in the `/data` directory:
-
-```yaml
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: pv0001
-spec:
-  accessModes:
-    - ReadWriteOnce
-  capacity:
-    storage: 5Gi
-  hostPath:
-    path: /data/pv0001/
+<a name="interacting-with-openshift"></a>
+## Interacting with OpenShift
+
+<a name="openshift-client-binary-oc"></a>
+### OpenShift client binary (oc)
+
+The `minishift start` command creates an OpenShift instance using the
```

Comment:
Totally agree @hferentschik and I suggested the same here in question 2: https://github.com/minishift/minishift/pull/473#issuecomment-281989882